projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1179,18 +1179,34 @@ public function get_post_content_for_csv_export( $post_id ) {
 	 * @return mixed
 	 */
 	public function get_post_meta_for_csv_export( $post_id ) {
-		$md                                        = get_post_meta( $post_id, '_feedback_extra_fields', true );
-		$md[ __( 'Date', 'jetpack-forms' ) ]       = get_the_date( DATE_RFC3339, $post_id );
-		$content_fields                            = self::parse_fields_from_content( $post_id );
-		$md[ __( 'IP Address', 'jetpack-forms' ) ] = ( isset( $content_fields['_feedback_ip'] ) ) ? $content_fields['_feedback_ip'] : 0;
+		$md                     = get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md['-3_response_date'] = get_the_date( 'Y-m-d H:i:s', $post_id );
+		$content_fields         = self::parse_fields_from_content( $post_id );
+		$md['93_ip_address']    = ( isset( $content_fields['_feedback_ip'] ) ) ? $content_fields['_feedback_ip'] : 0;
 
 		// add the email_marketing_consent to the post meta.
-		$md[ _x( 'Consent', 'noun', 'jetpack-forms' ) ] = 0;
+		$md['90_consent'] = 0;
 		if ( isset( $content_fields['_feedback_all_fields'] ) ) {
 			$all_fields = $content_fields['_feedback_all_fields'];
 			// check if the email_marketing_consent field exists.
 			if ( isset( $all_fields['email_marketing_consent'] ) ) {
-				$md[ _x( 'Consent', 'noun', 'jetpack-forms' ) ] = $all_fields['email_marketing_consent'];
+				$md['90_consent'] = $all_fields['email_marketing_consent'];
+			}
+			// check if the feedback entry has a title.
+			if ( isset( $all_fields['entry_title'] ) ) {
+				$md['-9_title'] = $all_fields['entry_title'];
+			}
+
+			// check if the feedback entry has a permalink we can use.
+			if ( isset( $all_fields['entry_permalink'] ) && ! empty( $all_fields['entry_permalink'] ) ) {
```

Comment:
You should be able to get rid of the `isset` here, the empty does the same check:

```suggestion
			if ( ! empty( $all_fields['entry_permalink'] ) ) {
```